### PR TITLE
yubico-piv-tool 2.1.0

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/yubico-piv-tool.rb [hunk 1]
```diff
@@ -13,14 +13,21 @@ class YubicoPivTool < Formula
...
+  depends_on "check"
   depends_on "openssl@1.1"
+  depends_on "pcsc-lite"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    ENV.append_to_cflags "-I#{Formula["pcsc-lite"].opt_include}/PCSC"
```
Comment (SMillerDev) [inline Formula/yubico-piv-tool.rb:26]: Is there no way to tell cmake this directly?

Comment (SeekingMeaning) [reply Formula/yubico-piv-tool.rb:26]: We could pass `-DCMAKE_C_FLAGS=-I#{Formula["pcsc-lite"].opt_include}/PCSC` to cmake

Comment (SMillerDev) [reply Formula/yubico-piv-tool.rb:26]: Sounds better to me.

**Review by SMillerDev — APPROVED** [review]
Thanks @SeekingMeaning  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**